applications/crossbar/src/crossbar_maintenance.erl

```diff
@@ -1182,3 +1184,23 @@ set_app_screenshots(AppId, PathToScreenshotsFolder) ->
               || SShot <- filelib:wildcard(kz_term:to_list(PathToScreenshotsFolder) ++ "/*.png")
              ],
     update_screenshots(AppId, MA, SShots).
+
+-spec db_init() -> 'ok'.
+db_init() ->
+    kz_datamgr:suppress_change_notice(),
+    _ = kz_datamgr:revise_doc_from_file(?KZ_CONFIG_DB, ?APP, <<"views/system_configs.json">>),
+    _ = kz_datamgr:revise_doc_from_file(?KZ_MEDIA_DB, ?APP, "account/media.json"),
+    _ = kz_datamgr:revise_doc_from_file(?KZ_RATES_DB, ?APP, <<"views/rates.json">>),
+    _ = kz_datamgr:revise_doc_from_file(?KZ_SIP_DB, ?APP, <<"views/resources.json">>),
+    _ = kz_datamgr:revise_doc_from_file(?KZ_PORT_REQUESTS_DB, ?APP, <<"views/port_requests.json">>),
+    _ = kz_datamgr:revise_doc_from_file(?KZ_ACDC_DB, ?APP, <<"views/acdc.json">>),
```

Comment:
I am not sure what to do about community apps having their assets referenced in kazoo's core apps. Like, when someone creates the next 'foobar' app with account views, does Crossbar now need a line added to load the foobar app's views and foobar needs to put their views in crossbar/priv?

Reply:
these were already there, but could/should be moved to the app that is responsible for it.
these apps should register/revide on app startup